### PR TITLE
Use vagrant-hostmanager and enable configuring the host url

### PR DESCRIPTION
**Pull request by m7moud**
Issue: https://github.com/spryker/spryker.github.io/issues/200

### CODE REVIEW

**Comment by dereuromark** [comment]
@marek-obuchowicz Please review this.

File: Vagrantfile [hunk 1]
```diff
@@ -5,6 +5,17 @@ def yellow(text); colorize(text, "\033[33m"); end
 def green(text); colorize(text, "\033[32m"); end
 def bold(text); colorize(text, "\033[1;97m"); end
 
+required_plugins = ['vagrant-vbguest', 'vagrant-hostmanager']
+plugins_to_install = required_plugins.select { |plugin| !Vagrant.has_plugin? plugin }
+unless plugins_to_install.empty?
+  puts "Installing plugins: #{plugins_to_install.join(', ')}"
+  if system "vagrant plugin install #{plugins_to_install.join(' ')}"
```
Comment (vlunov-spryker) [inline Vagrantfile]: So, if I got it correct, you are installing a plugin and running the same vagrant command one more time, why do you not exit in this main process then? Btw, what is the difference between `exec` and `system` here?

Comment (m7moud) [inline Vagrantfile]: Please refer to the comment there: https://github.com/spryker/devvm/pull/26/commits/b34f03c61b7ad54c349ddc4b9e71156d4d4ffb6c#diff-23b6f443c01ea2efcb4f36eedfea9089R15

**Comment by m7moud** [comment]
Closed in favour of https://github.com/spryker/devvm/pull/31

**Pull request closed without merging**